tool/grind.dart

```diff
@@ -14,9 +14,10 @@ void analyze() {
 }
 
 @Task()
-void test() {
-  Tests.runCliTests();
-}
+test() => Tests.runCliTests();
+
+@Task('Apply dartfmt to all Dart source files')
+format() => DartFmt.format(['bin', 'example', 'lib', 'test', 'tool', 'web']);
```

Comment:
Are you relating to `['bin', 'example', 'lib'...]`. This is intentional because with `'.'` also the `build` directory with all it's subdirectories (contains also copies of all deps) is formatted which is cumbersome and usually not desired.